runtime-display/jh7110-pvr-blobs/spec

```diff
@@ -0,0 +1,3 @@
+VER=1.19.6345021+git20231018
+SRCS="tbl::https://github.com/milkv-mars/mars-buildroot-sdk/raw/ff73df5dbe650cee7af609675d98146576a7f656/soft_3rdpart/IMG_GPU/out/img-gpu-powervr-bin-1.19.6345021.tar.gz"
```

Comment:
Fine... But any way to track for package updates? It seems that the `img-gpu-powervr-bin` package you specified matches with the part in `VER=` before `+git`, so why not just use that version?

Reply:
In the [commit history](https://github.com/starfive-tech/soft_3rdpart/commits/JH7110_VisionFive2_devel/IMG_GPU/out) of this file, there have been instances of the same version number with updated binary files, and it cannot be ruled out that similar situations may occur in the future.